process/elasticproc/updateTokenType.go

```diff
@@ -11,6 +11,13 @@ import (
 )
 
 func (ei *elasticProcessor) indexTokens(tokensData []*data.TokenInfo, updateNFTData []*data.NFTDataUpdate, buffSlice *data.BufferSlice) error {
+	if ei.isIndexEnabled(elasticIndexer.ESDTsIndex) {
```

Comment:
please extract common code & refactor as suggested in previous comment

Reply:
refactored.